examples/DVCLive-Quickstart.ipynb

```diff
@@ -0,0 +1,354 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"https://colab.research.google.com/github/iterative/dvclive/blob/main/examples/DVCLive-Quickstart.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

Comment:
This link would work once we merge on the main branch, in the meantime use  https://colab.research.google.com/github/iterative/dvclive/blob/colab/examples/DVCLive-Quickstart.ipynb